packages/utils/src/getLocalStorage.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Retrieves the value stored in localStorage for the given key.
+ * @param {string} key - The localStorage key.
+ * @returns {any} - The value stored in localStorage for the given key, or null if the key does not exist or has no value.
+ */
+export const getLocalStorage = (key: string) => {
+    const data = localStorage.getItem(key);
+    return data ? JSON.parse(data) : null;
+};
```

Comment:
In my opinion, this method is kinda redundant, It's not abstracting much. Maybe we can just simply use `localStorage.getItem` where we need it.

Reply:
Maybe, but I don't want to delete it in this PR :) Let's discuss it later

Reply:
@sergei-deriv if data is primitive, not JSON, this will throw the error '_Error: Unexpected end of JSON input_'. I guess this method was created specifically for processing objects (jsons) stored in localStorage. It should be documented here @mahdiyeh-deriv

Reply:
@maryia-deriv ok, I'll write like this:

`const data = localStorage.getItem(key) || '{}';`

Reply:
@maryia-deriv added check for this